### ISSUE TITLE
strange shapes with Thin and Regular weights

### ISSUE THREAD

**Issue by jdietrch** [marcelommp/Londrina-Typeface#7]
The picture below illustrates it well. I'm getting some strange shapes with the Thin and Regular weights of Londrina Solid. This is in InDesign CC 2018 on a Mac, but I see the same sort of problem in FontBook and LibreOffice too. I just installed the latest files (.otf) from master.

Is there something I'm not doing right? Or is there something I can do to help figure out what the problem is?

Thank you!
James

![screen shot 2018-07-11 at 2 33 51 pm](https://user-images.githubusercontent.com/1451525/42592587-f2e2ecde-8517-11e8-999e-53b4524c1c27.png)

**Comment by marcelommp** [comment]
I did some adjustments, but the problem is that Londrina has a lot of confuse nodes :)

https://github.com/marcelommp/Londrina-Typeface/tree/master/LastFontFiles

**Comment by jdietrch** [comment]
The problem still exists.

I checked the md5sum of `LondrinaSolid-Regular.otf`and it has not changed:
```
jdietrch@saturn:~/a/Londrina-Typeface [master]
$ git log --pretty=oneline -n 1
6d492b46f1d51a1c774a3974f87227fd30167ac1 Font Adjsutments
jdietrch@saturn:~/a/Londrina-Typeface [master]
$ md5sum LastFontFiles/OTF/LondrinaSolid-Regular.otf 
d853fa5ad226c6709d16c037e62db65a  LastFontFiles/OTF/LondrinaSolid-Regular.otf
jdietrch@saturn:~/a/Londrina-Typeface [master]
$ git checkout HEAD^
Note: checking out 'HEAD^'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a2b7062... Merge pull request #2 from m4rc1e/gf-mastering
jdietrch@saturn:~/a/Londrina-Typeface [(a2b7062...)]
$ md5sum OTF/LondrinaSolid-Regular.otf 
d853fa5ad226c6709d16c037e62db65a  OTF/LondrinaSolid-Regular.otf
jdietrch@saturn:~/a/Londrina-Typeface [(a2b7062...)]
$ 
```

**Comment by marcelommp** [comment]
I'm not too familiar with md5sum but I'm sure I've deleted a bunch of strange nodes before generate the last OTF and TTF files, but taking a big look again today I decided the regular and thin weights both will need a big revision. I'm reopening this issue to do this taks before the next release :)

**Comment by jdietrch** [comment]
Thanks for reopening this.

The md5sum thing simply means that the file LondrinaSolid-Regular.otf is identical to what it was before. You said you did some adjustments, but somehow those changes didn't make it to the otf file.

**Comment by marcelommp** [comment]
Can you test the other weights?

**Comment by jdietrch** [comment]
OK, I have looked at this some more, and I think I have a better idea of what's going on.

Several days ago, after you closed this issue saying that you had made some adjustments, I noticed right away that there were still strange shapes as originally reported. After that I noticed that the files in `OTF/` are identical to the files in `LastFontFiles/OTF/`, so of course they would have the same problems. Out of the seven files in `OTF/`, I observed problems with three of them:
`LondrinaOutline-Regular.otf` -- OK
`LondrinaShadow-Regular.otf` -- OK
`LondrinaSketch-Regular.otf` -- PROBLEM
`LondrinaSolid-Black.otf` -- OK
`LondrinaSolid-Light.otf` -- OK
`LondrinaSolid-Regular.otf` -- PROBLEM
`LondrinaSolid-Thin.otf` -- PROBLEM

There are, however, four additional files in `LastFontFiles/OTF/` that are not in `OTF/`:
```
LondrinaBlack-Regular.otf
LondrinaBook-Regular.otf
LondrinaSketche-Regular.otf
LondrinaThin-Regular.otf
```
And I now realize that it's these files that must contain the adjustments you did.

`LondrinaBlack-Regular.otf` -- OK, Looks like LondrinaSolid-Black.otf
`LondrinaBook-Regular.otf` -- OK, Looks like LondrinaSolid-Light.otf
`LondrinaSketche-Regular.otf` -- OK, Looks like a FIXED LondrinaSketch-Regular.otf
`LondrinaThin-Regular.otf` -- OK, Looks like a FIXED LondrinaSolid-Thin.otf
 I still don't see a fixed version of LondrinaSolid-Regular.otf, but I suppose that will come later.

So, thank you! Whatever you did looks like the right thing.
